config/datatypes_conf.xml.sample

```diff
@@ -367,6 +367,8 @@
       <display file="igv/vcf.xml"/>
       <converter file="vcf_bgzip_to_tabix_converter.xml" target_datatype="tabix"/>
     </datatype>
+    <datatype extension="ec" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true" />
```

Comment:
This and idx are very short extensions -- is there maybe an alternate to it ?

Reply:
idx is the output filetype of part of kallisto tools and input for another. I could add hidden_in_upload?

Reply:
That's not my concern, the extensions need to be unique across Galaxy and users should be able to tell what file it is based on the extension. How about kallisto.ec and kallisto.idx ?

Reply:
Ah. I can do that.